### PR TITLE
Cost widget created using the existing card component

### PR DESCRIPTION
**Pull request by fawadik**
Cost widget is created by re-using the card component and is displayed on storybook as well

### CODE REVIEW

**Review by Swapna-Sahu — COMMENTED** [review]
Component file is missing. Need to review again.

**Comment by fawadik** [comment]
> Component file is missing. Need to review again.

Created the required file

File: src/client/components/CostWidget.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import DataCard from "./DataCard";
+import { CardGroup, CardGroupHeader } from "./CardGroup";
+
+function CostWidget({ currentCost, remainingCost }) {
+  let totalCost = currentCost + remainingCost;
```
Comment (bytewiz) [inline src/client/components/CostWidget.js]: use `const` instead of `let` as the value is read-only anyway.

https://tylermcginnis.com/var-let-const/

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good to me! just that small change

File: src/client/components/CostWidget.js (hunk 1 above)
Comment (fawadik) [reply src/client/components/CostWidget.js]: changes made. replaced let with const

**Review by bytewiz — DISMISSED** [review]
Looks good to me! 👍

**Review by Swapna-Sahu — COMMENTED** [review]
Looks good.

File: src/client/stories/CostWidgetStory.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  const dataCard2 = number("Remaining Cost", 112.32);
+  const dataCard3 = dataCard1 + dataCard2;
+
+  return (
+    <div>
+      <CardGroupHeader title="Cost Widget" />
+      <CardGroup>
+        <DataCard title={titleCard1} data={"$" + dataCard1} />
+        <DataCard title={titleCard2} data={"$" + dataCard2} />
+        <DataCard title={titleCard3} data={"$" + dataCard3} />
```
Comment (Swapna-Sahu) [inline src/client/stories/CostWidgetStory.js:20]: Good.

**Review by Swapna-Sahu — APPROVED** [review]
Now looks good with "$" :)

**Comment by fawadik** [comment]
> Now looks good with "$" :)

Thanks alot for mentioning...!

**Review by zkwsk — APPROVED** [review]

**Comment by zkwsk** [comment]
Great work @fawadik! And thank you so much for helping with the review @Swapna-Sahu.

**Pull request merged**